sensor_msgs/Image image
std_msgs/Float32MultiArray point_range